app/kvsapp/source/kvsapp.c

```diff
@@ -947,6 +954,52 @@ int KvsApp_setoption(KvsAppHandle handle, const char *pcOptionName, const char *
                 pKvs->xStrategy.xRingBufferPara.uMemLimit = uMemLimit;
             }
         }
+        else if (strcmp(pcOptionName, OPTION_NETIO_CONNECTION_TIMEOUT) == 0)
```

Comment:
How about use `strncmp` instead to prevent `pcOptionName` is accidentally not end with `\0`?

Reply:
Sure, will do.

Reply:
If pcOptionName is not end with `\0`, then the comparison would end in the end of OPTION_NETIO_CONNECTION_TIMEOUT. And the result would be not zero since they don't have same length. I'll roll back this strncmp first.